frontend/src/views/Main/components/Sidebar/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import styles from "./SearchBar.module.css";
+
+const SearchBar = () => {
+	const [isfocused, setIsFocused] = useState(false);
+
+	const focus = (): void => {
```

Comment:
I think it might make sense here to just pass the setters for focus and unfocus inline in the jsx - less code and still self-explanatory enough. That way we can reserve this area for more of the heavy logic

Reply:
For sure!